.github/workflows/benchmark.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             - name: Setup Node
               uses: actions/setup-node@v3
               with:
-                  node-version: '18.20.1'
+                  node-version: 20
```

Comment:
Call me paranoid, but I tend to prefer putting exact version numbers in here. One less thing that might randomly start failing on some random day (especially with backport branches...)

Reply:
Yeah, I had an internal debate about this and settled on how it's probably not an issue since Node is super stable in my experience and v20 is LTE on top of that. I don't have a strong opinion on it though.